addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,42 +1,75 @@
 $gold: #eca801;
 $silver: #cccccc;
 $bronze: #eea91e;
+$o-wf-main-wrapper-width: map-get($container-max-widths, xl);
+$o-wf-main-wrapper-narrow-width: map-get($container-max-widths, md);
+$o-wf-sidebar-width: 16em !important;
+
 
 .website_forum {
-    .o_forum_ask_btn {
-        @include media-breakpoint-up(md) {
-            box-shadow: $box-shadow;
-            width: 200px;
-        }
-    }
+    --o-main-wrapper-width: #{$o-wf-main-wrapper-narrow-width};
+    --o-sidebar-width: #{$o-wf-sidebar-width};
 
-    .o_wforum_nav .nav-link {
-        @include o-hover-text-color($body-color, $link-color);
-        line-height: 1;
+    @include media-breakpoint-up(md){
+        --o-main-wrapper-width: #{$o-wf-main-wrapper-width};
+    }
 
-        .fa {
-            opacity: 0.5;
+    .o_wforum_main_wrapper {
+        @include media-breakpoint-up(md){
+            grid-template-columns: 1fr minmax(0, var(--o-main-wrapper-width)) 1fr;
+            gap: 1.2rem;
         }
+    }
 
-        &:hover, &.active {
-            .fa {
-                opacity: 1;
+    .o_wforum_sidebar {
+        width: var(--o-sidebar-width);
+        grid-template-rows: auto min-content;
+
+        .o_wforum_sidebar_section .nav-link{
+            border-radius: $border-radius;
+            &:hover {
+                background-color: $gray-200;
             }
         }
 
-        &.active {
-            background-color: rgba(map-get($theme-colors, 'info'), 0.1);
-            color: darken(map-get($theme-colors, 'info'), 15%)!important;
+        img.o_forum_avatar {
+            width: 50px;
+            height: 50px;
         }
+    }
 
+    .o_wforum_table {
         img.o_forum_avatar {
-            width: 30px;
-            height: 30px;
+            width: 36px;
+            height: 36px;
+            border: 2px solid var(--body-bg);
+        }
+
+        .o_wforum_reply_count, .o_wforum_view_count, .o_wforum_last_activity {
+            width: 100px;
+        }
+
+        tbody tr:hover {
+            background-color: darken($body-bg, 2%);
+        }
+    }
+
+    .o_wforum_index_entry_title a{
+        &:link {
```

Comment:
Indeed, I tend to forget about that. 
This was a FP request though, should I set a `text-reset` instead then ?

Reply:
```html
<a><h5>Title</h5></a> 
```
?

Reply:
`h5` (or any of the other headings) doesn't have a defined color. Could I maybe set the `a` font color to `dark` and then lighten the `dark` when the link is visited ? It was important for FP that there be a strong difference to visited links.

Reply:
I'm not against the behaviour, actually I think that's an usability improvement.
But it shouldn't be done in this way, targeting inconsistently forum links only (and not all of them, just a few 😢).
I propose a patch in `website` to replicate a similar behaviour globally ... to be tested

Reply:
I agree, keeping in mind that the patch should allow for the link's color to be the same as the text-color/different to primary and secondary.
Should I remove this change and put a note in the PR?

Reply:
I think you can do it in this same PR, in `website`, ideally with the use of bootstrap variables.
Let's check if @pierre-pde agrees first

Reply:
@stefanorigano & @edi-odoo ,
I looked at two websites: Amazon & Wikipedia.
I clicked a few link and didn't notice a different of text color on the previously clicked links.
So, I wouldn't want this as a standard website feature.

However, if FP wants this for forum (and I can understand why), I think we should do it there.
Ideally clicked link should be link but darker.

I'm open to discussion on this :)

Reply:
Hey there @pierre-pde ,

> I clicked a few link and didn't notice a different of text color on the previously clicked links.

Browsers have limited the ability to do this kind of thing for privacy reasons, so that's probably why you don't see it on regular websites, [read about it here](https://hacks.mozilla.org/2010/03/privacy-related-changes-coming-to-css-vistited/)

However I do think it's interesting for a page such as a forum or a blog, because, as FP said, you want to know what you've already read either to go back to it or to avoid it. (I know I do that with google links which turn purple when you've visited them)

> Ideally clicked link should be link but darker.

In this particular case though, FP wants the links to be initially black, so making them darker when visited isn't really an option ;) 

I've made this a custom feature just for forum for now. I still think it might be interesting for blog. WDYT?